cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -44,6 +46,7 @@ func NewCmd(o *Options) *cobra.Command {
 	cobraCmd.Flags().DurationVarP(&o.QuitTimeout, "quit-timeout", "", 1200*time.Second, "Time after which the deployment is aborted. Worker goroutines may still be working in the background. This value must be greater than the value for cancel-timeout.")
 	cobraCmd.Flags().DurationVarP(&o.HelmTimeout, "helm-timeout", "", 360*time.Second, "Timeout for the underlying Helm client.")
 	cobraCmd.Flags().IntVar(&o.WorkersCount, "workers-count", 4, "Number of parallel workers used for the deployment.")
+	cobraCmd.Flags().BoolVarP(&o.Verbose, "verbose", "v", false, "Enable verbose mode.")
```

Comment:
There is already a verbose flag on kyma level https://github.com/kyma-project/cli/blob/master/cmd/kyma/kyma.go#L52. Could you please reuse it instead of creating a new one?